node/cn/tracers/api.go

```diff
@@ -76,7 +76,7 @@ const (
 )
 
 var (
-	HeavyAPIRequestLimit int32 = 500
+	HeavyAPIRequestLimit int32 = 500 // WARN: changing this value will have no effect. This value is for test. See HeavyDebugRequestLimitFlag
```

Comment:
The comment seems confusing to me. Modification of this value seems to have an effect, incorporating the value of `HeavyDebugRequestLimitFlag.`

Reply:
The value is always overriden by this line:
https://github.com/klaytn/klaytn/blob/1f02226a32b040e57a6aa19e43b27718869dbe3f/cmd/utils/config.go#L700

Reply:
Then can we go the other way around?

```go
HeavyDebugRequestLimitFlag = &cli.IntFlag{
    Value: tracers.DefaultHeavyAPIRequestLimit,
}

var (
    DefaultHeavyAPIRequestLimit int32 = 500
    HeavyAPIRequestLimit = DefaultHeavyAPIRequestLimit // The default is primarily used for testing. When the node starts, it will be overridden by HeavyDebugRequestLimitFlag (--rpc.unsafe-debug.heavy-debug.request-limit)
)
```

Reply:
Reflected 1805e974 cc @yoomee1313